### PR TITLE
added Benson portfolio to engineers folder

### PR DESCRIPTION
**Pull request by WTF-ATuan**

### CODE REVIEW

File: src/engineering.html [hunk 1]
```diff
@@ -307,6 +307,22 @@ <h3><a href="https://zfo.gg/">Zachary Fogg</a></h3>
...
+          <a href="https://www.linkedin.com/in/benson-tsai-vr-developer/">
+            <div class="thumb">
+              <div class="avatar-container">
+                <div class="avatar-head"></div>
+                <div class="avatar-body"></div>
+              </div>
+            </div>
+          </a>
+          <div class="bio-info">
+            <h3><a href="https://zfo.gg/">Benson Tsai</a></h3>
```
Comment (nooma-42) [inline src/engineering.html]: Is this a wrong link?

Comment (WTF-ATuan) [reply src/engineering.html]: yes! sorry I foogot to change that.

**Review by komali2 — CHANGES_REQUESTED** [review]
Looks really good, but please change the first-person to the 3rd person. e.g. instead of "i'm a VR developer," "Benson is a VR developer..." "he's worked on projects such as..." etc

**Comment by WTF-ATuan** [comment]
I already change the depiction!

**Review by komali2 — APPROVED** [review]

**Pull request merged**